testsuite/features/core_proxy_branch_network.feature

```diff
@@ -121,6 +121,12 @@ Feature: Setup SUSE Manager for Retail branch network
     When I enable repositories before installing branch server
     And I install package "expect" on this "proxy"
 
+@proxy
+@private_net
+  Scenario: Let avahi work on the branch server
+    Given service "firewalld" is active on "proxy"
+    When I open avahi port on the proxy
```

Comment:
Will this work? Firewalld is not yet running at this stage. Branch network activates it on the highstate run. At least that was the reason I moved it after running highstate. For this to work then https://github.com/uyuni-project/uyuni/pull/870/files#diff-cd82af202d0a443fa4c8d00547ec32e5R627 would need to use `firewall-offline-cmd` command IMO.

Reply:
@aaannz That's precisely the problem :smile_cat: .

It has to go before running highstate, because after highstate the communication is already cut and the game is over. I cant  send any more commands via ssh at this point.

But before highstate, firewalld is not running yet.

The way to solve this chicken and egg problem is :
 1 - to move it before highstate
 2 - to start firewalld by hand OR to use this `firewall-offline-cmd` command.

I'll implement it ASAP, as it prevents me (or anyone else) to test on QA servers or on workstations.